### PR TITLE
by Ak

### PR DESCRIPTION
**Pull request by y-u-m-iy-u-m-i**

### CODE REVIEW

File: .github/workflows/ci-cd.yml [hunk 1]
```diff
@@ -54,6 +58,39 @@ jobs:
...
+          cp ./backup.sh /backup.sh
+
+          CRON_JOB="/backup.sh"
+          output=$(crontab -l | grep -c "$CRON_JOB")
+          if [ ! $output -eq 0 ]; then
+              echo "Cron job already exists."
+          else
+              (crontab -l; echo "*/50 * * * * /backup.sh") | crontab -
+              echo "Cron job added."
+          fi
```
Comment (abdullahalhoothy) [inline .github/workflows/ci-cd.yml:92]: please isolate this in a seperate file instead of inline script

File: .github/workflows/ci-cd.yml [hunk 2]
```diff
@@ -54,6 +58,39 @@ jobs:
...
+
           # Build and run Docker containers
-          docker-compose build
-          docker-compose up -d
\ No newline at end of file
+          docker compose build
+          docker compose up -d
```
Comment (abdullahalhoothy) [inline .github/workflows/ci-cd.yml:96]: shouldn't this be "docker-compose" ?

File: 123secrets_variables.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "DATABASE_URL": "postgresql: //scraper_user:scraper_password@db:5432/aqar_scraper",
+    "DATABASE_HOST": "db",
+    "POSTGRES_DB": "aqar_scraper",
+    "POSTGRES_USER": "scraper_user",
+    "POSTGRES_PASSWORD": "scraper_password",
+    "PGADMIN_DEFAULT_EMAIL": "email@e.com",
+    "PGADMIN_DEFAULT_PASSWORD": "pass",
+    "PGADMIN_CONFIG_SERVER_MODE": "True",
+    "PGADMIN_CONFIG_MASTER_PASSWORD_REQUIRED": "False"
```
Comment (abdullahalhoothy) [inline 123secrets_variables.json:10]: this is correct to be False? didn't we say we need password?

**Review by abdullahalhoothy — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-cd.yml (hunk 2 above)
Comment (ahmed-kamal2004) [reply .github/workflows/ci-cd.yml:96]: docker-compose is an old version written in python, so it causes problems with env variables
While docker compose is newer and it doesn't have problems
i tested both of them.

Comment (ahmed-kamal2004) [reply .github/workflows/ci-cd.yml:96]: docker compose is also a plugin for docker written in go

File: 123secrets_variables.json (hunk 3 above)
Comment (ahmed-kamal2004) [reply 123secrets_variables.json:10]: Yes I made it true for
PGADMIN_CONFIG_SERVER_MODE
But for PGADMIN_CONFIG_MASTER_PASSWORD_REQUIRED
It is another thing about a master password for initialization

Now the pgadmin is configured as we  talked

File: .github/workflows/ci-cd.yml (hunk 1 above)
Comment (ahmed-kamal2004) [reply .github/workflows/ci-cd.yml:92]: Ok

**Review by abdullahalhoothy — APPROVED** [review]

**Pull request merged**